### PR TITLE
Style update for Books

### PR DESCRIPTION
**Pull request by Bibref**
The bibliography needed an update for books because of a punctuation problem. Adding a `<choose>` with two different outcomes will solve this problem

### CODE REVIEW

**Comment by rmzelle** [comment]
You're missing a `</else-if>` at the end (just before `<else-if type="patent">`).

You can update your pull request by using the edit (pencil) icon at https://github.com/Bibref/styles/blob/patch-4/ets-ecole-de-technologie-superieure.csl

**Comment by zuphilip** [comment]
@rmzelle : There is an update. @Bibref : There are no automatic notifications for commits, thus you should write a short note here as well next time.

File: ets-ecole-de-technologie-superieure.csl [hunk 1]
```diff
@@ -163,21 +163,42 @@
...
+                    <else-if match="any" variable="page">
+                      <text variable="page" suffix=" p"/>
+                    </else-if>
+                  </choose>
+                </group>
+              </group>
+            </if>
+            <else>
+              <group delimiter=". " prefix=". ">
+                <text macro="edition"/>
```
Comment (rmzelle) [inline ets-ecole-de-technologie-superieure.csl]: @Bibref, why would you use the "edition" macro here, if the "edition" variable is empty here?

**Comment by Bibref** [comment]
I deleted the macro "edition" as it was not necessary.

**Comment by rmzelle** [comment]
It would be better to use:

```
        <else-if type="book" match="any">
          <text macro="edition" prefix=", "/>
          <group delimiter=". " prefix=". ">
            <text macro="collection"/>
            <group delimiter=", ">
              <text macro="publisher"/>
              <choose>
                <if match="any" variable="number-of-pages">
                  <text variable="number-of-pages" suffix=" p"/>
                </if>
                <else-if match="any" variable="page">
                  <text variable="page" suffix=" p"/>
                </else-if>
              </choose>
            </group>
          </group>
        </else-if>
```

by the way, since then you don't duplicate as much code. (not tested, but I think I got it right)

**Comment by Bibref** [comment]
Very good suggestion. Changes were made!

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**